libtcmu_log.c

```diff
@@ -209,6 +209,34 @@ static void log_output(int pri, const char *msg, bool bypass)
 	}
 }
 
+struct log_buffer {
+	int pri;
+	char msg[LOG_MSG_LEN];
+};
+
+static darray(struct log_buffer) log_tmpbuf = darray_new();
+static pthread_mutex_t log_tmpbuf_lock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
Can you move all these definitions to the top of the file with the rest of them?

Reply:
Sure.